### PR TITLE
Set up personas

### LINKED ISSUE #393: Set up personas
Create organisations, users, and other resources we'll use for thinking up new features.
This will include seeds that we might be creating in our staging environment.

### PR DESCRIPTION
**Pull request by almirsarajcic**
Resolves #393.

### CODE REVIEW

**Comment by almirsarajcic** [comment]
@wintermeyer, can you check this one when you have some time?

File: lib/omedis/accounts/changes/new_activity_position.ex [hunk 1]
```diff
@@ -16,7 +16,14 @@ defmodule Omedis.Accounts.Changes.NewActivityPosition do
         max_position =
           Activity
           |> Ash.Query.filter(group_id == ^group_id)
-          |> Ash.count!(authorize?: false, tenant: organisation)
+          |> Ash.Query.select(:position)
```
Comment (wintermeyer) [inline lib/omedis/accounts/changes/new_activity_position.ex:19]: Code is fine. Maybe this or something similar is easier to read?

```elixir
max_position =
  Activity
  |> Ash.Query.filter(group_id == ^group_id)
  |> Ash.Query.aggregate(:max_position, max(:position))
  |> Ash.read_one!(authorize?: false, tenant: organisation)
  |> Map.get(:max_position, 0)
```

File: priv/repo/personas_seeds.exs [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+
+# Events for Spitex Bemeda
+%{records: _medical_events, status: :success} =
+  bulk_create.(
+    Accounts.Event,
+    [
+      %{
+        activity_id: visit_patient.id,
+        user_id: denis.id,
+        dtstart: ~U[2024-03-20 08:00:00Z],
```
Comment (wintermeyer) [inline priv/repo/personas_seeds.exs:401]: I am not a big fan of using fixed dates in these seeds. I suggest that we use the current date and move forward or backward from that.

**Review by wintermeyer — DISMISSED** [review]
@almirsarajcic I don't like the fixed dates in the seeds. But that is your call. We can change that any time.

**Comment by almirsarajcic** [comment]
Thanks

**Review by wintermeyer — APPROVED** [review]

**Pull request merged**